packages/strapi-admin/admin/src/assets/images/logo-admin.svg

```diff
@@ -0,0 +1 @@
-.5-3.91V20.86h-4.41Z"/><path class="cls-1" d="M145.71,21.42h-1.28a11.17,11.17,0,0,0-4.74.89,7.2,7.2,0,0,0-3,2.57V42.06H129V14.89h7.37v4.8A7.54,7.54,0,0,1,138.88,16a7.16,7.16,0,0,1,4.55-1.34,8.37,8.37,0,0,1,2.28.28Z"/><path class="cls-1" d="M163.23,39.16a13.53,13.53,0,0,1-3.63,2.45,11.32,11.32,0,0,1-4.9,1,8.58,8.58,0,0,1-3.24-.59A7,7,0,0,1,149,40.41,7.46,7.46,0,0,1,147.44,38a8.26,8.26,0,0,1-.56-3.07,6.76,6.76,0,0,1,2.52-5.69A12.19,12.19,0,0,1,156.26,27c1.11-.08,2.27-.13,3.46-.17l3.23-.11v-.95a5.64,5.64,0,0,0-1.28-4,5.3,5.3,0,0,0-4-1.37,15.35,15.35,0,0,0-4,.51,14.9,14.9,0,0,0-3.35,1.28l-1.51-5.36A18,18,0,0,1,153.47,15a23.09,23.09,0,0,1,5.52-.64c3.61,0,6.44,1,8.51,3s3.09,4.89,3.09,8.76v16h-7.36Zm-5.75-1.73a8.11,8.11,0,0,0,3-.56A10,10,0,0,0,163,35.48v-4.3l-2.51.22c-.89.08-1.71.17-2.45.28a7.37,7.37,0,0,0-2.68.92,2.36,2.36,0,0,0-1,2.15,2.39,2.39,0,0,0,.87,2A3.73,3.73,0,0,0,157.48,37.43Z"/><path class="cls-1" d="M176.23,14.89h7.31v3.46a9.91,9.91,0,0,1,3.34-2.93,10.15,10.15,0,0,1,4.91-1.09,9.86,9.86,0,0,1,4.47,1,10.19,10.19,0,0,1,3.43,2.81,13.49,13.49,0,0,1,2.2,4.35,19.34,19.34,0,0,1,.78,5.67,16.32,16.32,0,0,1-1.11,6.22,13.27,13.27,0,0,1-3,4.54,12.35,12.35,0,0,1-4.43,2.77,15.93,15.93,0,0,1-5.44.92,16.37,16.37,0,0,1-2.68-.2,19.31,19.31,0,0,1-2.12-.47v11.6h-7.64Zm13,5.41a6.36,6.36,0,0,0-3.21.78,7.72,7.72,0,0,0-2.15,1.73V36.26a8,8,0,0,0,1.67.41,13,13,0,0,0,2,.14c2.61,0,4.5-.8,5.69-2.39a10.13,10.13,0,0,0,1.79-6.2,10.54,10.54,0,0,0-1.42-5.66A4.81,4.81,0,0,0,189.23,20.3Z"/><path class="cls-1" d="M206.13,7a4.36,4.36,0,0,1,1.23-3.13,4.21,4.21,0,0,1,3.18-1.28A4.34,4.34,0,0,1,215,7a4.3,4.3,0,0,1-1.26,3.09,4.21,4.21,0,0,1-3.2,1.32,4.13,4.13,0,0,1-3.18-1.32A4.35,4.35,0,0,1,206.13,7Zm.61,35.09V14.89h7.65V42.06Z"/><path class="cls-1" d="M30.77,5,26,2.08,1.93,16,2,20l9.11,5.26L17,21.84l1.3-6.32a1,1,0,0,1,.47-.64l21-12.07a1,1,0,0,1,1,0l15.64,9a1,1,0,0,1,.48.83l0,18.1a1,1,0,0,1-.48.83L35.58,43.52a1,1,0,0,1-.74.09l-6.3-1.78-5.9,3.43V55.71l3.45,2L50.16,43.78v-5.6l1.92-1.26v7.41a1,1,0,0,1-.48.83l-25,14.49a1,1,0,0,1-.48.13,1,1,0,0,1-.48-.13L21.19,57.1a1,1,0,0,1-.48-.83V44.71a1,1,0,0,1,.48-.83L27.92,40a1,1,0,0,1,.75-.09L35,41.65,54.87,30.23l0-17L40.23,4.76,20.13,16.33l-1.3,6.33a1,1,0,0,1-.46.64l-6.81,3.89a1,1,0,0,1-1,0L.54,21.39a1,1,0,0,1-.48-.83L0,15.47a1,1,0,0,1,.48-.84L25.54.13a1,1,0,0,1,1,0l6.16,3.74Z"/><path class="cls-1" d="M41.56,26.29a.94.94,0,0,1-.48-.13l-3.82-2.22a1,1,0,0,1-.48-.83v-4.4a1,1,0,0,1,.48-.84l3.82-2.2a1,1,0,0,1,1,0l3.83,2.19a1,1,0,0,1,.48.83v4.44a1,1,0,0,1-.48.83L42,26.16A.89.89,0,0,1,41.56,26.29Zm-2.85-3.74,2.86,1.67,2.85-1.65V19.25l-2.87-1.64L38.7,19.26Z"/><path class="cls-1" d="M11.55,33.74A1,1,0,0,1,11.07,32L14.42,30a1,1,0,0,1,1,1.66L12,33.61A.91.91,0,0,1,11.55,33.74Z"/><path class="cls-1" d="M10.84,39.57a1,1,0,0,1-.48-1.79l6.42-3.68a1,1,0,0,1,1.31.35,1,1,0,0,1-.35,1.32l-6.43,3.67A.9.9,0,0,1,10.84,39.57Z"/><path class="cls-1" d="M11.88,44.35a1,1,0,0,1-.84-.49,1,1,0,0,1,.37-1.31l7.75-4.35a1,1,0,1,1,.94,1.68l-7.75,4.34A.9.9,0,0,1,11.88,44.35Z"/></svg>
```

Comment:
Hi @jasongitmail thanks a lot for your contribution. However, I was wondering if you could make your PR against the develop branch where `node-sass` as been removed?

Thanks again!